devenv/lib/context.py

```diff
@@ -0,0 +1,14 @@
+from __future__ import annotations
+
+from typing import TypeAlias
+from typing import TypedDict
+
+from devenv.lib.repository import Repository
+
+Path: TypeAlias = str
+
+
+class Context(TypedDict):
```

Comment:
might be nice to change this to an actual class rather than a dict -- up to you!